prismic-model/json/editorial-contributor-roles.json

```diff
@@ -1,11 +1,18 @@
 {
-  "Contributor" : {
-    "title" : {
-      "type" : "StructuredText",
-      "config" : {
-        "single" : "heading1",
-        "label" : "title"
+  "Contributor": {
+    "title": {
+      "type": "StructuredText",
+      "config": {
+        "single": "paragraph,hyperlink,strong,em,heading1",
+        "label": "Title"
+      }
+    },
+    "describedBy": {
+      "type": "Text",
+      "config": {
+        "label": "Byline word for role",
```

Comment:
`Word describing the role [in the byline]`

Adding the byline ties it to design, which could change and makes editorial people feel like they're controlling design, where it's content that should portable. 

I am also being 💎 about it.

Reply:
Yeah, makes sense.